### PR TITLE
Refocus: Cleaning up libraries

### PR DESCRIPTION
**Pull request by zklosko**
This PR hopes to start 2021 with some (slightly early) Spring Cleaning by removing unused libraries and files.

Removals include

- TableTools (incl. Flash parts, exporting in Playout History)
- Widgets hidden from main interface
- References to Airtime Pro in the interface and code for Airtime Pro monitoring features
- Extra CSS
- Extra uncalled JS scripts

Doesn't fully address #249 yet, but gets us halfway there.

Update: I've decided to change goals. Instead of focusing on converting objects to Vue, I plan to focus on simplifying the JavaScript objects and libraries being used in order to make maintenance and rewriting to Vue easier. I'd like this to be considered before we release 3.0.0 alpha 9.

### CODE REVIEW

**Comment by paddatrapper** [comment]
How does this affect #1122?

**Comment by zklosko** [comment]
It doesn't affect #1122. If anything, it gives @hairmare something to copy/paste logic from instead of rewriting the entire thing from scratch.

Also assuming the rewrite will be a lengthy process, this will help us fix some of the issues we've had and clean up some clutter without having to continue to use outdated frameworks and libraries.

**Comment by hairmare** [comment]
#1122 also contains very much config and code related to making it possible to use new Vue views in the legacy apps as drop in replacements for old interfaces w/o touching routing & logic. It's main difference from this is that it is aiming at having these new views consume data from the django-rest-framework API rather than the legacy backend.

All in all #1122 is pretty much at the state when it's gonna depend on pull requests like API 2.0 #958 and GitHub Actions #1125 soon.

**Comment by zklosko** [comment]
@hairmare okay.

I noticed that my pull request was getting a little big, so I'm deciding to trim it down a little bit. The new home request is going to be focused more around removing libraries that we don't use. There are a lot of them.

I would also like to start removing muses in favor of howler.js so the players work a little bit better without a flash fallback. It would be nice if this could go in the 3.0.9 alpha release.

File: install [hunk 1]
```diff
@@ -1210,6 +1210,7 @@ else
     # not on centos
     ip_device="eth0"
     $is_ubuntu_xenial && ip_device="enp0s8"
+    $is_ubuntu_bionic && ip_device="enp0s2"
```
Comment (hairmare) [inline install:1213]: I don't really Ubuntu so I can't say if this makes sense.

**Review by hairmare — COMMENTED** [review]
I like the refocus of this 👍 It LGTM, i'll give it a final clean-room install test and then approve & merge it!

File: install (hunk 1 above)
Comment (zklosko) [reply install:1213]: It doesn't even work, TBH 😝

**Review by hairmare — APPROVED** [review]

**Pull request merged**